cmd/lakectl/cmd/log.go

```diff
@@ -100,6 +114,10 @@ var logCmd = &cobra.Command{
 		if amountForPagination <= 0 {
 			amountForPagination = internalPageSize
 		}
+		// case --no-merges & --amount, ask for more results since some will filter out
+		if noMerges && amountForPagination < maxAmountNoMerges {
+			amountForPagination *= 3
```

Comment:
Now I get the `333` - it's directly linked to 3. I wonder how our linter doesn't complain about this magic number.